db/version_builder.cc

```diff
@@ -403,14 +403,14 @@ class VersionBuilder::Rep {
 
               return Status::Corruption("VersionBuilder", oss.str());
             }
-          } else if (lhs->fd.smallest_seqno <= rhs->fd.smallest_seqno) {
-            std::ostringstream oss;
-            oss << "L0 file #" << lhs->fd.GetNumber() << " with seqno "
-                << lhs->fd.smallest_seqno << ' ' << lhs->fd.largest_seqno
-                << " vs. file #" << rhs->fd.GetNumber() << " with seqno "
-                << rhs->fd.smallest_seqno << ' ' << rhs->fd.largest_seqno;
-
-            return Status::Corruption("VersionBuilder", oss.str());
+            // } else if (lhs->fd.smallest_seqno <= rhs->fd.smallest_seqno) {
```

Comment:
Is it possible to ease the check instead of commenting it out?

Reply:
Changing to `<` should work.